src/main/resources/db/migration/V28__svarid_søknadstabeller.sql

```diff
@@ -0,0 +1,71 @@
+
+ALTER table Soknadsskjema
+    ADD COLUMN bosituasjon_deler_du_bolig_verdi VARCHAR;
+ALTER table Soknadsskjema
+    ADD COLUMN bosituasjon_deler_du_bolig_svar_id VARCHAR;
```

Comment:
Må vi slette `bosituasjon_deler_du_bolig` kolonnen som finnes fra før?

Reply:
I en ideell verden må man egentlig utføre dette i to separate brancher/PR-er (for bakoverkompabilitet)
Branch 1:
- Opprette den nye kolonnen
- Kopier data fra den gamle kolonnen over til den nye

Branch 2:
- Slett den gamle kolonnen


Siden vi ikke har data i prod er det ikke krise å gjøre dette i samme branch, men vil anbefale å kopiere data over samt slette den gamle kolonnen.